src/screens/SpeedUpAndCancelSheet.js

```diff
@@ -230,6 +232,9 @@ export default function SpeedUpAndCancelSheet() {
     } catch (e) {
       logger.log('Error submitting cancel tx', e);
     } finally {
+      if (isHardwareWallet) {
+        goBack();
+      }
```

Comment:
`goBack` is called twice in these blocks?

Reply:
we need an extra one to close the hardware tx sheet